.rubocop_u2i.yml

```diff
@@ -0,0 +1,56 @@
+require: rubocop-rspec
+
+AllCops:
+  RunRailsCops: true
+  Include:
+    - '**/Rakefile'
+    - config.ru
```

Comment:
No, for `.rubocop_u2i.yml` I want to have it the same everywhere.
Maybe we could split it into `.rubocop_u2i.yml` and `.rubocop_rails_u2i.yml`?